modules/k8s-nmstate-troubleshooting-dns-disconnected-env-resolv.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-troubleshooting-node-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="k8s-nmstate-troubleshooting-dns-disconnected-env-resolv.adoc_{context}"]
+= Creating a custom DNS host name to resolve DNS connectivity issues
+
+In a disconnected environment where the external DNS server cannot be reached, you can resolve Kubernetes NMState Operator health probe issues by specifying a custom DNS host name in the `NMState` custom resource definition (CRD). 
+
+.Procedure
+
+* Add the DNS host name configuration to the `NMState` CRD of your cluster:
++
+[source,terminal]
+----
+apiVersion: nmstate.io/v1
+kind: NMState
+metadata:
+  name: nmstate
+spec:
+  probeConfiguration:
+    dns:
+      host: redhat.com
+# ...
+----
+
```

Comment:
Not to get out of scope and provide technical review, but would the customer need to run `oc create -f <filename>.yaml` or `oc apply -f <filename>.yaml`? I think that is how it usually works for CRDs.

If this change will be applied automatically, I think it would be a good idea to mention that.

Reply:
I am not sure what is meant by "this change will be applied automatically", but usually how we (in Engineering) see it is

1) you create a yaml file somewhere in your system, we don't care where, can be a file in your desktop
2) you run `oc apply -f path-to-the-file`
3) done

The issue with `oc create [...]` is that it does **not** allow to modify a resource that already exists. So it is desired to use `apply`, not `create`

Reply:
Thank you for clarifying!

Reply:
Thanks. I added the additionally step about applying the configuration to the cluster network. Unless I should state to apply the configuration to nodes?